src/test/regress/expected/isolation_hash_copy_vs_all.out

```diff
@@ -97,13 +97,12 @@ create_distributed_table
 step s1-initialize: COPY hash_copy FROM PROGRAM 'echo 0, a, 0\\n1, b, 1\\n2, c, 2\\n3, d, 3\\n4, e, 4' WITH CSV;
 step s1-begin: BEGIN;
 step s1-copy: COPY hash_copy FROM PROGRAM 'echo 5, f, 5\\n6, g, 6\\n7, h, 7\\n8, i, 8\\n9, j, 9' WITH CSV;
-step s2-insert-select: INSERT INTO hash_copy SELECT * FROM hash_copy; <waiting ...>
+step s2-insert-select: INSERT INTO hash_copy SELECT * FROM hash_copy;
```

Comment:
Do we have any tests that checks that we'll wait when replication factor > 1? If not, we should add some.